### DISCUSSION TITLE
Rekono is available on Kali Linux

### DISCUSSION THREAD

**Discussion by pablosnt** [pablosnt/rekono#247] [Announcements]
Rekono Desktop has been added to [Kali Linux](http://pkg.kali.org/pkg/rekono-kbx) with the name `rekono-kbx`. You can install it by executing `apt install rekono-kbx`, and then running `rekono`. Check it out!

**Comment by zinwelzl** [comment]
This is great!

Can you add API testing tools?

**Reply by pablosnt** [reply]
What do you mean by API testing tools?

**Comment by zinwelzl** [comment]
https://owasp.org/www-community/api_security_tools
https://github.com/arainho/awesome-api-security
Or search internet for more open-source tools.

Currently most users use REST API, if you can add that tools first.
There is more API specs.

**Reply by pablosnt** [reply]
Oh sorry, sure! I will select some tools to detect vulnerabilities in APIs, and I will add them to Rekono. Thank you for the proposal! 

I'm not sure yet, but this tools are likely to be added in the 2.0.0 version that is going to take me some time, as it's a breaking version.

**Comment by zinwelzl** [comment]
Thanks!
